web-ng/src/app/components/card-view-project/card-view-project.component.css

```diff
@@ -0,0 +1,25 @@
+.project-card {
+    background-image: url(/assets/img/happy-land.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    max-width: 400px;
+    padding: 32px;
+    }
+
+  
+    
+  .logo {
+      width: 64px;
+      height: 64px;
+   }
+  
+  .title-project{
+      width: 100px;
+      color: #fff;
```

Comment:
Please convert this file to scss and use themed colors rather than color literals here and throughout. See other .scss files for examples.